### PR TITLE
update(push-to-registries.yml): use build/buildx-matrix

### PR DESCRIPTION
**Pull request by ACoolmanTelicent**
Testing https://github.com/Telicent-io/telicent-test/pull/145

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ jobs:
   publish_deployment_image_to_registries:
     needs:
       - get_version
-    uses: Telicent-io/telicent-test/.github/workflows/push-to-registries.yml@main
+    uses: Telicent-io/telicent-test/.github/workflows/push-to-registries.yml@build/buildx-matrix
```
Comment (Greg-teli100) [inline .github/workflows/publish.yml:23]: Is this a temporary change until https://github.com/Telicent-io/telicent-test/pull/145 is merged?

Also, this is a public repo is the intention for this to use the public/oss workflows and repos?

Comment (ACoolmanTelicent) [reply .github/workflows/publish.yml:23]: Yeah, this is temporary so I can test the branch `build/buildx-matrix` with this repo.

I've not triggered the `.github/workflows/publish.yml` workflow yet. 

I'm giving some time for people to have a look.


> Is the intention for this to use the public/oss workflows and repos?

Hm. I guess its the default approach. I wish there was a simpler solution tbh.

Comment (ACoolmanTelicent) [reply .github/workflows/publish.yml:23]: [Rob also doesn't like it](https://github.com/Telicent-io/docs/blob/main/eng/scripts/dx/about-system.sh)

Comment (ACoolmanTelicent) [reply .github/workflows/publish.yml:23]: [Rob V also doesn't love it](https://github.com/Telicent-io/telicent-test/pull/145#pullrequestreview-1986532160)

**Comment by ACoolmanTelicent** [comment]
[PR unneeded](https://github.com/Telicent-io/telicent-test/pull/145#issuecomment-2051582220)

**Pull request closed without merging**